core/src/avm2/api_version.rs

```diff
@@ -0,0 +1,63 @@
+use gc_arena::Collect;
+
+// Based on https://github.com/adobe/avmplus/blob/master/core/api-versions.h
+// FIXME - if we ever add in AIR emulation, then implement the 'version series' logic.
+// For now, it's fine to just compare Flash player version against air versions directly.
+#[derive(Copy, Clone, Collect, PartialEq, Eq, PartialOrd, Ord, Hash, FromPrimitive, Debug)]
+#[collect(require_static)]
```

Comment:
If we can, let's put `#[collect(require_static)]` on struct fields instead of making trivial types GC-aware.